lua/Report.lua

```diff
@@ -93,6 +93,29 @@ Report_ = {
 		mandatoryArgument(1, "string", text)
 		self.text[self.nextIdx_] = text
 	end,
+	
+
+	-- Add a new tabela to the Report.
+	-- @arg text A mandatory string with the tabela to the report.
+	-- @usage import("publish")
+	-- local report = Report()
+	-- report:addText("My tabela")
+
+	addTable = function(self, data)   --Nova function para criação de tabelas no modal do Report
+		self.tabela[self.nextIdx_] = data
```

Comment:
Translate tabela to English

Reply:
in this case I need to use other name, the name (table) conflicts with another variable of the same name already existing in the system.